src/components/ReviewPage.jsx

```diff
@@ -1,20 +1,31 @@
 import { useEffect, useState } from "react";
 import { useParams } from "react-router-dom";
 import { fetchReviewById } from "../utils/api";
+import { fetchComments } from "../utils/api";
+import CommentList from "./CommentList";
 import Loading from "./Loading";
 import Vote from "./Vote";
 
 const ReviewPage = () => {
   const [isLoading, setIsLoading] = useState(false);
   const [review, setReview] = useState({});
   const { review_id } = useParams();
+  const [comments, setComments] = useState([]);
+
   useEffect(() => {
     setIsLoading(true);
     fetchReviewById(review_id).then(({ review }) => {
       setReview(review);
       setIsLoading(false);
     });
   }, [review_id]);
+
+  useEffect(() => {
+    fetchComments(review_id).then(({ comments }) => {
+      setComments(comments);
+    });
+  });
```

Comment:
You're missing a dependency array for this. At the moment, its constantly doing this request and re-rendering it. 

Would this not work in the same useEffect as above?